### PR TITLE
fix publish warnings

### PR DESCRIPTION
**Pull request by danylo-safonov-solid**
```
% dart pub publish --dry-run          
Resolving dependencies...
  _fe_analyzer_shared 61.0.0 (64.0.0 available)
  analyzer 5.13.0 (6.2.0 available)
  analyzer_plugin 0.11.2 (0.11.3 available)
  dart_code_metrics 5.7.5 (5.7.6 available)
  dart_code_metrics_presets 1.8.0 (2.4.0 available)
  dart_style 2.3.2 (2.3.3 available)
  file 6.1.4 (7.0.0 available)
  http 0.13.6 (1.1.0 available)
  process 4.2.4 (5.0.0 available)
  pub_updater 0.3.0 (0.3.1 available)
  uuid 3.0.7 (4.0.0 available)
Got dependencies!
Publishing deno_postgres_interop 0.0.1 to https://pub.dev:
├── CHANGELOG.md (<1 KB)
├── LICENSE (1 KB)
├── README.md (1 KB)
├── add_imports.yaml (<1 KB)
├── analysis_options.yaml (<1 KB)
├── bin
│   └── add_imports.dart (1 KB)
├── example
│   ├── analysis_options.yaml (<1 KB)
│   ├── lib
│   │   └── main.dart (1 KB)
│   └── pubspec.yaml (<1 KB)
├── lib
│   ├── deno_postgres_interop.dart (<1 KB)
│   └── src
│       ├── add_imports
│       │   ├── args.dart (<1 KB)
│       │   └── config.dart (1 KB)
│       ├── client.dart (<1 KB)
│       ├── isolation_level.dart (<1 KB)
│       ├── query_client.dart (2 KB)
│       ├── query_object.dart (1 KB)
│       ├── query_object_result.dart (<1 KB)
│       ├── transaction.dart (1 KB)
│       ├── transaction_options.dart (1 KB)
│       └── util.dart (<1 KB)
└── pubspec.yaml (<1 KB)
Validating package...

Package has 0 warnings.
The server may enforce additional checks.
```

### CODE REVIEW

**Comment by danylo-safonov-solid** [comment]
@illia-romanenko I'm not sure tho, the `add_imports` part will only be used as a dev dependency.
But right now, when package will be depended on, the use will also have dependencies of the package in their main dependencies, not dev ones

File: pubspec.yaml [hunk 1]
```diff
@@ -8,5 +8,10 @@ environment:
   # Tested with 3.0.6, but others should work too
   sdk: '>=3.0.0 <3.1.0'
 
+dependencies:
+  args: ^2.4.2
+  collection: ^1.18.0
+  yaml: ^3.1.2
```
Comment (danylo-safonov-solid) [inline pubspec.yaml:14]: I'm talking about that. These packages are only used in add_imports, which is only used as a dev script

**Comment by danylo-safonov-solid** [comment]
Should we ~~have it as~~move it to a separate package instead?

**Review by illia-romanenko — APPROVED** [review]
LGTM - when you are ready.

**Pull request merged**